demos/browser/app/meeting/meeting.ts

```diff
@@ -550,7 +551,12 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
   }
 
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    const logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
+    let logger: MeetingSessionPOSTLogger | ConsoleLogger = null;
```

Comment:
You should pass the logger type from the server in JoinInfo structure to avoid localhost check below.

Reply:
PR 162 is making the change

Reply:
In that case you should not touch this code. And it also makes deployment safer.